Classes/Backend/Decorator/InterfaceDecorator.php

```diff
@@ -30,7 +30,7 @@
 /**
  * Decorator interface.
  *
- * @method void __construct(tx_rnbase_mod_IModule $mod)
+ * @method void __construct(\tx_rnbase_mod_IModule $mod)
```

Comment:
I think this should be `\Sys25\RnBase\Backend\Module\IModule`, shouldn't it?

Reply:
Updated and repushed.